From 3ed91c638a134529d437e5c241bb7bc2dc4bcd90 Mon Sep 17 00:00:00 2001
From: Pan Xiuli <xiuli.pan@linux.intel.com>
Date: Tue, 10 Jul 2018 10:51:25 +0800
Subject: [PATCH 1321/1676] ASoC: Intel: fix dai link for bxt_pcm512x

Change cpu dai name for SSP5 and add capture capability.

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>
(cherry picked from commit 675767ce6a2a7dadc0ad305915ed776622aad7a1)
---
 sound/soc/intel/boards/bxt_pcm512x.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/sound/soc/intel/boards/bxt_pcm512x.c b/sound/soc/intel/boards/bxt_pcm512x.c
index 5bc8085..9fe6bb4 100644
--- a/sound/soc/intel/boards/bxt_pcm512x.c
+++ b/sound/soc/intel/boards/bxt_pcm512x.c
@@ -105,7 +105,7 @@ static struct snd_soc_dai_link dailink[] = {
 	{
 		.name = "SSP5-Codec",
 		.id = 0,
-		.cpu_dai_name = "sof-audio",
+		.cpu_dai_name = "SSP5 Pin",
 		.platform_name = "sof-audio",
 		.no_pcm = 1,
 		.codec_dai_name = "pcm512x-hifi",
@@ -116,6 +116,7 @@ static struct snd_soc_dai_link dailink[] = {
 		.be_hw_params_fixup = codec_fixup,
 		.nonatomic = true,
 		.dpcm_playback = 1,
+		.dpcm_capture = 1,
 	},
 };
 
-- 
2.7.4

